pykickstart/commands/ignoredisk.py

```diff
@@ -151,7 +153,29 @@ class F29_IgnoreDisk(F14_IgnoreDisk):
     removedAttrs = F14_IgnoreDisk.removedAttrs
 
     def parse(self, args):
-        return F8_IgnoreDisk.parse(self, args)
+        retval = FC3_IgnoreDisk.parse(self, args)
+
+        howmany = 0
+        if self.ignoredisk:
+            howmany += 1
+        if self.onlyuse:
+            howmany += 1
+        if self.interactive:
+            howmany += 1
+            warnings.warn(_("Ignoring deprecated option on line %s:  The ignoredisk command "
+                            "no longer supports --interactive option.  In future releases, "
+                            "this will result in a fatal error from kickstart. "
+                            "Please modify your kickstart file to remove the option.")
+                          % self.lineno, KickstartDeprecationWarning)
+        if howmany != 1:
```

Comment:
What about removing the `howmany` and change this line to:

```suggestion
        if sum(map(int, (self.ignoredisk, self.onlyuse, self.interactive))) != 1:
```

Reply:
Yes, howmany seems a bit funny to me, but I prefer to keep it consistent with other code.

Reply:
OK, just an idea :). You know in deep of my hearth I like functional programming :D.